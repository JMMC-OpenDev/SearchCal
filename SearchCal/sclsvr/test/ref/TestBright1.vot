version https://git-lfs.github.com/spec/v1
oid sha256:59320251add807392ca1f0a0b9ca0c5074d0ddd664273f2a2ba40c314b89e756
size 3301269
