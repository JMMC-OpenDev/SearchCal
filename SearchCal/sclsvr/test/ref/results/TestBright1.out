version https://git-lfs.github.com/spec/v1
oid sha256:faf28b21d9f6dfed9e39413251485f7a498ccd61de90fc8139403cc2c05f7676
size 1530956
